Packs/HealthCheck/Scripts/HealthCheckSystemDiagnostics/HealthCheckSystemDiagnostics.py

```diff
@@ -141,7 +142,7 @@ def FormatTableAndSet(data, dataSource):
 
 results = CommandResults(
     readable_output="HealthCheck System Diagnostics Done",
-    outputs_prefix="HealthCheck.ActionableItems",
+    outputs_prefix="dbstatactionableitems",
```

Comment:
This will break BC.
I don't think it is a big deal in this pack use case but just raising awareness.

Reply:
its not a change, its a fix, otherwise no action items will be presented in the table